draft-ietf-quic-http.md

```diff
@@ -909,7 +913,7 @@ peer will assume that the connection has been closed.  HTTP/3 implementations
 will need to open a new HTTP/3 connection for new requests if the existing
 connection has been idle for longer than the idle timeout negotiated during the
 QUIC handshake, and SHOULD do so if approaching the idle timeout; see Section
-10.1 of {{QUIC-TRANSPORT}}.
+10.1.1 of {{QUIC-TRANSPORT}}.
```

Comment:
```suggestion
10.1 of {{QUIC-TRANSPORT}}.
```
10.1.1 deals with transport-level liveness testing before sending data, which isn't contemplated here.

Reply:
FWIW I proposed 10.1.1 specifically for "SHOULD [use a new connection] if approaching the idle timeout", which I did not see discussed elsewhere in the section.  But I will revert it back.